README.md

```diff
@@ -44,20 +44,20 @@ pip install --editable .
 ## Script Usage
 
 ```sh
-exhibitor-tls-artifacts [OPTIONS] [SANS]...
+Usage: exhibitor-tls-artifacts [OPTIONS] [NODES]...
+
+  Generates Admin Router and Exhibitor TLS artifacts. NODES should consist
+  of a space seperated list of master ip addresses. See
```

Comment:
Do these have to be IP addresses? IPv4? Or can these also be DNS names?

Reply:
@mhrabovcin Found a place in the docs that instructed the users to configure exhibitor to use ip addresses for communication. I guess these could be ipv6 addresses as well. I will defer to Martin on this one.

Reply:
The version of this tool should support customers with static masters. From our documentation

> master_list A YAML nested list (-) of static master IP addresses.

https://docs.mesosphere.com/1.13/installing/production/advanced-configuration/configuration-reference/#exhibitor-storage-backend
https://docs.mesosphere.com/1.13/installing/production/advanced-configuration/configuration-reference/#master-discovery-required

Reply:
Okay cool.

Looks like currently master_list only supports IPv4 values: https://github.com/dcos/dcos/blob/9eee5cdb1fefbb9e64c630776baa383fdb53414e/gen/calc.py#L413